### PR TITLE
Request power usage for xiaomi power plug

### PR DESCRIPTION
**Pull request by hvanoch**
Enable requesting power usage for the xiaomi ZNCZ04LM

Fixes https://github.com/Koenkk/zigbee2mqtt/issues/4146

### CODE REVIEW

File: converters/toZigbee.js [hunk 1]
```diff
@@ -1168,6 +1168,13 @@ const converters = {
             return {state: {power_outage_memory: value}};
         },
     },
+    xiaomi_power: {
+        key: ['power'],
+        convertGet: async (entity, key, meta) => {
+            const endpoint = meta.device.getEndpoint(21);
```
Comment (Koenkk) [inline converters/toZigbee.js:1174]: This line worries me a bit, it makes the converter specific to this device (as the other Xiaomi devices might not have this endpoint.

Can you try changing it to:

```js
const endpoint = meta.device.endpoints.find(e => e.supportsInputCluster("genAnalogInput"));
```

File: devices.js [hunk 2]
```diff
@@ -868,7 +868,7 @@ const devices = [
             fz.ignore_occupancy_report,
             fz.ignore_illuminance_report, fz.ignore_time_read,
         ],
-        toZigbee: [tz.on_off, tz.xiaomi_switch_power_outage_memory],
+        toZigbee: [tz.on_off, tz.xiaomi_power, tz.xiaomi_switch_power_outage_memory],
```
Comment (Koenkk) [inline devices.js:871]: Please also add tz.xiaomi_power to the other devices having fz.xiaomi_power (like I did in https://github.com/Koenkk/zigbee-herdsman-converters/pull/1497/files)

Comment (hvanoch) [reply devices.js:923]: Done

File: converters/toZigbee.js (hunk 1 above)
Comment (hvanoch) [reply converters/toZigbee.js]: Done

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**